### PR TITLE
[IRRC-92] Create Edit Teacher Form

### PR DESCRIPTION
**Pull request by lauratsui**
Frontend changes to enable edit Teacher.
Dependent on some changesi n #72, and also pending the set up of a patch in vuex

### CODE REVIEW

**Review by aelbore — APPROVED** [review]
Looks good

**Review by nondescryptid — APPROVED** [review]
form looks good, can merge this once the merge conflicts in router.js are resolved

**Comment by aelbore** [comment]
@lauratsui @szktm i manually resolved the conflicts

**Pull request merged**
